### PR TITLE
Add quotes

### LINKED ISSUE #181: Martinize needs quotes
There are no fun quotes at the end of a martinize run. This is extremely sad.

### PR DESCRIPTION
**Pull request by pckroon**
Added some initial quotes. Most are still (too) Martini specific though.

Fixes #181

### CODE REVIEW

File: vermouth/data/quotes.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+A man must defend his home, his wife, his children, and his martini. -- Jackie Gleason
+
+I never go jogging, it makes me spill my martini. -- George Burns
+Shaken, not stirred. -- James Bond
```
Comment (jbarnoud) [inline vermouth/data/quotes.txt]: For consistency with the rest of the file, you could skip a line between these two quotes.

File: vermouth/data/quotes.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Hearts full of youth! Hearts full of truth! Six parts gin to one part vermouth. -- Tom Lehrer (Bright College Days)
+
+First and foremost, before we continue I'd like to make one thing perfectly clear: A martini is made with gin. If your martini is made with vodka, it is not, in fact, a martini. -- Sterling Archer (How To Archer)
+
+The only proper way to make a martini is with ice-cold gin and a bow in the direction of France. -- Winston Churchill
+
+A perfect Martini should be made by filling a glass with gin, and then waving it in the general direction of Italy. -- Noël Coward
+
+A martini should always be stirred, not shaken, so that the molecules lie sensuously on top of one another. -- Somerset Maugham
```
Comment (jbarnoud) [inline vermouth/data/quotes.txt]: I really like this one :smile:

**Comment by pckroon** [comment]
I like the idea of only showing a quote when there's no warning. That makes this PR more complex though, and that should probably be separated out into another PR. Maybe also together with a `-maxwarn` and not writing output files.

**Comment by jbarnoud** [comment]
Can I squash all the commits into one?

**Comment by pckroon** [comment]
Yes

**Pull request merged**